src/components/Routes/RouterSet.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {lazy, Suspense} from 'react';
+import Loader from "../Loader/Loader.jsx";
+import {Routes, Route} from "react-router-dom";
+import PrivateRoute from "./PrivateRoute.jsx";
+import RestrictedRoute from "./RestrictedRoute.jsx";
+import Layout from "../Layout/Layout.jsx";
+
+
+const HomePage = lazy(() => import("../../pages/HomePage/HomePage.jsx"));
+const LoginPage = lazy(() => import("../../pages/LoginPage/LoginPage.jsx"));
+const RegisterPage = lazy(() => import("../../pages/RegisterPage/RegisterPage.jsx"));
+const UploadPhoto = lazy(() => import("../../components/UploadPhoto/UploadPhoto.jsx"));
+const ArticlesPage = lazy(() => import("../../pages/ArticlesPage/ArticlesPage.jsx"));
+const ArticlePage = lazy(() => import("../../pages/ArticlePage/ArticlePage.jsx"));
+const AuthorsPage = lazy(() => import("../../pages/AuthorsPage/AuthorsPage.jsx"));
+const AuthorsProfilePage = lazy(() => import("../../pages/AuthorProfilePage/AuthorProfilePage.jsx"));
+const CreateArticlePage = lazy(() => import("../../pages/CreateArticlePage/CreateArticlePage.jsx"));
+
+
+const RouterSet = () => {
+
+
+    return (
+        <Suspense fallback={<Loader/>}>
+            <Routes>
+                <Route path="/" element={<Layout/>}>
+                    <Route index element={<HomePage/>}/>
+                    <Route path="/create" element={<PrivateRoute component={<CreateArticlePage/>} redirectTo="/login"/>}/>
```

Comment:
тут потрібно компонент CreateArticlePage обгорнути PrivateRoute
```suggestion
                    <Route path="/create" element={<PrivateRoute>
                <CreateArticlePage />
              </PrivateRoute>}/>
```